src/commands/distribute/stores/category.txt

```diff
@@ -0,0 +1 @@
+Commands to distribute to stores (App Store, Google Play, Intune)
```

Comment:
Remove "Commands to" to follow convention. Consider adding an object, the thing that you can distribute.

Reply:
I see releases does this 'commands for' thing. It doesn't add anything though, just takes up space.

Reply:
nvm let's keep what you wrote until we do another doc cleanup

Reply:
I see this is a unique situation in the CLI where a "command folder" (distribute) has both commands (release) and more command folders (groups, releases, stores) inside it. So the sentence makes sense.

Reply:
I agree but we can deal with these inconsistencies in another PR.